devDocs/startupShutdownInstall.md

```diff
@@ -0,0 +1,93 @@
+# NVDA Starting and Exiting outline
+
+## Ways to start NVDA:
+
+1. For an installed copy:
+    1. Ctrl+Alt+N (Desktop shortcut)
+    1. Automatically via Ease of Access on the Windows sign-in screen (at boot or signing out of a previous session)
+    1. Automatically via Ease of Access on User Account Control (UAC) screens
+    1. Automatically by Ease of Access after signing in to Windows
+1. For an installed copy, portable copy, installer:
+    1. An exiting instance of NVDA starting a new process (see shutting down procedures)
+    1. By running the exe (see cli options too).
+    This can be triggered by a user or external process such as an existing NVDA instance
+    - test: `startupShutdownNVDA.Starts`
+1. For source: eg runnvda.bat
+
+## NVDA can be shutdown by:
+
+1. UI within NVDA, with and without an ExitDialog prompt (uses `triggerNVDAExit`):
+    1. NVDA+q
+        - test: `startupShutdownNVDA.Quits from keyboard, Restarts`
+    1. An unbound command to restart

```

Comment:
What does this mean? Is it referring to a command that by default has no gesture bound to it?